static/app/views/dashboards/widgetBuilder/components/widgetBuilderSlideout.tsx

```diff
@@ -49,6 +51,7 @@ function WidgetBuilderSlideout({
   isWidgetInvalid,
 }: WidgetBuilderSlideoutProps) {
   const {state} = useWidgetBuilderContext();
+  const [initialState, _setInitialState] = useState(state);
```

Comment:
```suggestion
  const [initialState] = useState(state);
```
I think you can just do this to only grab the initial state